src/storages/types.ts

```diff
@@ -118,16 +126,52 @@ export interface ICustomStorageWrapper {
    * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a list.
    */
   getItemsCount: (key: string) => Promise<number>
+
+  /** Set operations */
+
   /**
-   * Returns if item is a member of a list.
+   * Returns if item is a member of a set.
    *
    * @function itemContains
-   * @param {string} key list key
+   * @param {string} key set key
    * @param {string} item item value
-   * @returns {Promise<boolean>} A promise that resolves with true boolean value if `item` is a member of the list stored at `key`, or false if it is not a member or `key` list does not exist.
-   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a list.
+   * @returns {Promise<boolean>} A promise that resolves with true boolean value if `item` is a member of the set stored at `key`, or false if it is not a member or `key` set does not exist.
+   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a set.
    */
   itemContains: (key: string, item: string) => Promise<boolean>
+  /**
+   * Add the specified `items` to the set stored at `key`. Those items that are already part of the set are ignored.
+   * If key does not exist, an empty set is created before adding the items.
+   *
+   * @function addItems
+   * @param {string} key set key
+   * @param {string} items items to add
+   * @returns {Promise<void>} A promise that resolves if the operation success.
+   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a set.
+   */
+  addItems: (key: string, items: string[]) => Promise<void>
+  /**
+   * Remove the specified `items` from the set stored at `key`. Those items that are not part of the set are ignored.
+   *
+   * @function removeItems
+   * @param {string} key set key
+   * @param {string} items items to remove
+   * @returns {Promise<void>} A promise that resolves if the operation success. If key does not exist, the promise also resolves.
+   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a set.
+   */
+  removeItems: (key: string, items: string[]) => Promise<void>
+  /**
+   * Returns all the items of the `key` set.
+   *
+   * @function getItems
+   * @param {string} key set key
+   * @returns {Promise<string[]>} A promise that resolves with the list of items. If key does not exist, the result is an empty list.
+   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a set.
```

Comment:
Typo: `... if there is a connectin error ` => ` if there is a connection error `